Description: Fix FTBFS on hurd due to missing sys/mount.h
Author: Ryan Kavanagh <rak@debian.org>
Origin: Debian
Applied-Upstream: https://github.com/poolpOrg/OpenSMTPD/commit/47c2fb22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: opensmtpd/configure.ac
===================================================================
--- opensmtpd.orig/configure.ac	2013-09-10 06:13:18.739419748 -0400
+++ opensmtpd/configure.ac	2013-09-10 06:13:18.735419638 -0400
@@ -261,6 +261,7 @@
 	sys/ndir.h \
 	sys/pstat.h \
 	sys/statfs.h \
+	sys/mount.h \
 	sys/un.h \
 	ucred.h \
 	util.h \
Index: opensmtpd/smtpd/queue_fsqueue.c
===================================================================
--- opensmtpd.orig/smtpd/queue_fsqueue.c	2013-09-09 20:40:36.000000000 -0400
+++ opensmtpd/smtpd/queue_fsqueue.c	2013-09-10 06:13:57.476485902 -0400
@@ -27,7 +27,9 @@
 #include <sys/statfs.h>
 #endif
 #include <sys/param.h>
+#if HAVE_SYS_MOUNT_H
 #include <sys/mount.h>
+#endif
 
 #include <ctype.h>
 #include <err.h>
